### PR TITLE
Update SassEngine to handle node-sass 2.x return value.

### PR DESCRIPTION
**Pull request by gabooh**
node-sass changed its API starting with 2.0.0-beta as seen here https://github.com/sass/node-sass/releases/tag/v2.0.0-beta

The renderSync return value is now an object and the rendered result is in {return value}.css.

### CODE REVIEW

File: lib/mincer/engines/sass_engine.js [hunk 1]
```diff
@@ -65,11 +65,16 @@ function sassError(ctx /*, options*/) {
...
+    var result = sass.renderSync({
       data:         this.data,
       includePaths: [ path.dirname(this.file) ].concat(context.environment.paths),
       indentedSyntax: /^.*\.sass$/.test(this.file)
     });
+    if (result.css) {
+      this.data = result.css;
+    } else {
+      this.data = result;
+    }
```
Comment (ixti) [inline lib/mincer/engines/sass_engine.js]: Please, shorten this to:

``` javascript
this.data = result.css || result;
```

Or actually check `node-sass` version instead ;))

Comment (gabooh) [inline lib/mincer/engines/sass_engine.js]: I though of checking the version, but did not find a way to do this.
I could check sass.info (it's new in >v2.0.0). But it's the same as checking for result.css... :)

Comment (puzrin) [inline lib/mincer/engines/sass_engine.js]: https://github.com/sass/node-sass#success

IMHO it costs nothing to add sourcemap support. It it possible to do all at once?

Comment (gabooh) [inline lib/mincer/engines/sass_engine.js]: After reading the code in coffee_engine.js, adding source map support looks like adding `this.map = result.map;`

I've not tried this yet.

Comment (puzrin) [inline lib/mincer/engines/sass_engine.js]: https://github.com/nodeca/mincer/wiki/How-to-create-addon

Comment (gabooh) [inline lib/mincer/engines/sass_engine.js]: Thanks for the hint. Maybe I'll take a look later with another pull request.

**Comment by Barnabas** [comment]
+1 Would appreciate this PR to use connect-assets. Thanks!

**Comment by OverFlow636** [comment]
+1 also would like, thanks!

**Comment by steel** [comment]
+1

**Comment by ixti** [comment]
Closed via 0d374f8026ffaa0cded6858eb51c0fa65373b918.
Thanks!

**Comment by gabooh** [comment]
Thanks :)
